src/index.css

```diff
@@ -0,0 +1,227 @@
+  text-transform: uppercase;
+  font-weight: 500;
+  padding: 10px 0;
+}
+
+nav a::after {
+  content: '';
+  position: absolute;
+  left: 0;
+  right: 0;
+  bottom: -5px;
+  height: 2px;
+  background-color: #00acdc;
+  display: none;
+}
+
+nav a:hover,
+nav a.is-active {
+  transition: all 300ms ease;
+  color: #00acdc;
+}
+
+nav a:hover::after,
+ul > li:nth-of-type(1) > a::after {
+  transition: all 300ms ease;
+  display: block;
+}
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  height: 408px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card:hover {
+  transition: all 300ms ease;
+  transform: scale(1.2);
+}
+
+.card__image {
+  background-image: url(./images/imac.jpeg);
+  background-size: contain;
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 160px;
+  height: 134px;
+  margin: 33px 17px;
+}
+
+.card__title {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  text-align: left;
+  color: #060b35;
+  width: 166px;
+  height: 36px;
+  margin-left: 17px;
+}
+
+.card:hover .card__title {
+  transition: all 300ms ease;
+  color: #34568b;
+}
+
+.card__code {
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  text-align: left;
+  color: #616070;
+  margin-left: 17px;
+  margin-bottom: 15px;
+}
+
+.card__reviews-wrapper {
+  display: flex;
+  width: 166px;
+  margin-left: 17px;
+  margin-bottom: 20px;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.card__reviews {
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  text-align: right;
+}
+
+.card__price-info {
+  display: flex;
+  width: 166px;
+  justify-content: space-between;
+  align-items: center;
+  margin-left: 17px;
+  margin-bottom: 20px;
+}
+
+.card__text {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 400;
+  line-height: 18px;
+  text-align: left;
+}
+
+.card__price {
+  font-family: Roboto, sans-serif;
+  font-size: 16px;
+  font-weight: 700;
+  line-height: 18px;
+  text-align: right;
+}
+
+.card__button {
+  width: 166px;
+  height: 40px;
+  margin-bottom: 17px;
+  margin-left: 17px;
+  background-color: #00acdc;
+  border-radius: 5px;
+}
+
+.card__button:hover {
+  transition: all 300ms ease;
+  background-color: #fff;
+  border: 1px solid #00acdc;
+}
+
+.card__button-text {
+  font-family: Roboto, sans-serif;
+  font-size: 14px;
+  font-weight: 700;
+  line-height: 16px;
+  text-align: center;
+  color: #fff;
+}
+
+.card__button-text:hover {
+  transition: all 300ms ease;
+  color: #00acdc;
+}
+
+.star {
+  display: inline-block;
+  background-image: url(./images/star.svg);
+  width: 13px;
+  height: 12px;
+  border-radius: 0.5px;
+}
+
+.stars--4 > .star:nth-child(1),
+.stars--4 > .star:nth-child(2),
+.stars--4 > .star:nth-child(3),
+.stars--4 > .star:nth-child(4) {
+  background-image: url(./images/star-active.svg);
+}
```

Comment:
The class name `.stars--4` should be reviewed for BEM consistency. Modifier classes in BEM typically use double hyphens, but the number '4' as a modifier might not be descriptive enough. Consider a more descriptive modifier name.